docs/source/models/concept_models/vivarium_mncnh_portfolio/facility_model.rst

```diff
@@ -0,0 +1,97 @@
+
+Coming up with values for these correlations and conditional probabilities that are consistent with GBD and external evidence is detailed at the end of this document.  But before we get to that complexity, let's start with how we will use these correlations and conditional probabilities in the simulation.
+
+[[Causal model diagram goes here]]
+
+Correlated propensities
+-----------------------
+
+This section describes how we will model an "instrisic correlation" of ANC, home delivery, and LBWSG.  In short, we will use a Gaussian copula to model this, which has three parameters capturing the correlation between each pair of the three propensities.
+
+The motivation for these correlations is as follows: we hypothesize that there are important "common causes" that are not shown explicitly in the diagram above.  For example, having a home delivery and having no ANC visits might both be influenced by rurality --- if all health services are offered far away, it is logical that people will be able to access them less.
+Similarly, it is likely that there are social exclusion factors causeing both exposure to LBWSG risk and lack of access to ANC and in-facility birth.
+In a simulation model where we have not included scenarios that change these common-cause factors, we do not have to model their effects explicitly.
+For our purposes, it is sufficient to capture the correlations between ANC, in-facility birth, and LBWSG risk exposure.
+
+In Vivarium, we use values selected uniformly at random from the interval [0,1], which we call propensities, to keep attributes like LBWSG and ANC calibrated at the population level.  This makes it straightforward to represent the correlation in our factors by generating correlated propensities. The `statsmodels.distributions.copula.api.GaussianCopula` implementation can make them::
+
+    from statsmodels.distributions.copula.api import GaussianCopula
+    copula = GaussianCopula([[1., .5, .1],
+                             [.5, 1., .1],
+                             [.1, .1, 1.]])
+    copula.rvs(10)
+
+.. list-table:: Propensity Correlation Stand-in Parameters
+   :header-rows: 1
+   :widths: 20 20 20
+
+   * - Factor A
+     - Factor B
+     - Correlation
+   * - ANC Propensity
+     - In-facility Birth Propensity
+     - 0.50
+   * - ANC Propensity
+     - LBWSG Propensity
+     - 0.10
+   * - In-facility Birth Propensity
+     - LBWSG Propensity
+     - 0.10
+
+Eventually we must put draws of consistent values in the artifact.
+
+Special ordering of LBWSG categories
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When we calibrate the model, it will help to have an ordering of the LBWSG categories that makes them most likely to correlate with the ANC and home delivery propensities.  To achieve this, we will order them from lowest average RR to highest average RR (averaged across all draws).  [[Perhaps more details to come, such as code or an explicit table.]]
```

Comment:
> it will help to have an ordering of the LBWSG categories that makes them most likely to correlate with the ANC and home delivery propensities

This phrase is confusing to me -- I'm not sure what you're trying to say.

Reply:
> To achieve this, we will order them from lowest average RR to highest average RR (averaged across all draws). [[Perhaps more details to come, such as code or an explicit table.]]

We need to specify that this ordering is sex-specific (so there's one ordering for males and a different ordering for females), and uses the RRs from the **early neonatal** age group since we're interested in quantifying the risk _at birth_.

Reply:
See if my edits work better for you